oauth-rest-server/pom.xml

```diff
@@ -23,11 +23,22 @@
             <version>${project.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>de.adorsys.xs2a.adapter</groupId>
+            <artifactId>xs2a-adapter-service-remote</artifactId>
+            <version>0.0.4-SNAPSHOT</version>
```

Comment:
I'd propose to keep the versions of the dependencies somewhere in the properties (like we have with it with `javax.servlet-api`). The same stays for `spring-webmvc` from line 120.
What do you think?

Reply:
done